Gating ctest on Fedora latest is failing on a ssg python module test

It appears the test started failing by itself, no changes made in the code could indicate something like this. I'll continue investigating. It fails only on latest Fedora container. Other distros seem to be ok.

~~This could be related https://github.com/ComplianceAsCode/content/pull/10411 but the gating for that pull request worked just fine in this same gating test.~~

For example, from: https://github.com/ComplianceAsCode/content/actions/runs/4600335639/jobs/8126802809

we have
```

___________________________ test_rule_to_xml_element ___________________________

rule_accounts_tmout = <ssg.build_yaml.Rule object at 0x7fbb87617290>

    def test_rule_to_xml_element(rule_accounts_tmout):
        xmldiff_main = pytest.importorskip("xmldiff.main")
        rule_el = rule_accounts_tmout.to_xml_element()
        with temporary_filename() as real:
            ET.ElementTree(rule_el).write(real)
            expected = os.path.join(DATADIR, "accounts_tmout.xml")
>           diff = xmldiff_main.diff_files(real, expected)

../../tests/unit/ssg-module/test_build_yaml.py:446: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/usr/local/lib/python3.11/site-packages/xmldiff/main.py:51: in diff_files
    return _diff(
/usr/local/lib/python3.11/site-packages/xmldiff/main.py:37: in _diff
    return diff_trees(
/usr/local/lib/python3.11/site-packages/xmldiff/main.py:27: in diff_trees
    return list(diffs)
/usr/local/lib/python3.11/site-packages/xmldiff/diff.py:435: in diff
    etree.register_namespace(k, v)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

>   ???
E   ValueError: Prefix format reserved for internal use
```

--- a/.github/workflows/gate.yaml
+++ b/.github/workflows/gate.yaml
@@ -135,7 +135,7 @@ jobs:
       - name: Install Deps
         run: dnf install -y cmake make openscap-utils python3-pyyaml python3-setuptools python3-jinja2 bats python3-pytest python3-pytest-cov ansible python3-pip ShellCheck git python3-mypy python3-lxml
       - name: Install deps python
-        run: pip install ruamel.yaml yamlpath types-openpyxl types-PyYAML xmldiff==2.5 openpyxl openpyxl-stubs pandas
+        run: pip install ruamel.yaml yamlpath types-openpyxl types-PyYAML xmldiff openpyxl openpyxl-stubs pandas
       - name: Checkout
         uses: actions/checkout@v2
       - name: Build

